### PR TITLE
refactored Player methods <organisation

### PR DESCRIPTION
**Pull request by MatthijsRijlaarsdam**

### CODE REVIEW

**Comment by wouterz** [comment]
Chopping up big methods like update and handleCollision improves readability and prevents having really long methods. <organisation

File: core/src/main/java/sem/group15/bubblebobble/core/LevelParser.java [hunk 1]
```diff
@@ -21,18 +21,7 @@
...
-     0 (enemy type), 200(x location), 100(y location)
-     Wall (object type), 20(x location), 100(y location)
-     Wall (object type), 200(x location), 100(y location)
-     Wall (object type), 200(x location), 100(y location)
-     Wall (object type), 200(x location), 100(y location)
-     Floor (object type), 200(x location), 100(y location)
-     Floor (object type), 200(x location), 100(y location)
-     Floor (object type), 200(x location), 100(y location)
+     (object type, location.x, location.y)
      */
```
Comment (DaanVermunt) [inline core/src/main/java/sem/group15/bubblebobble/core/LevelParser.java]: Clean to fix this <layout>

File: core/src/main/java/sem/group15/bubblebobble/core/objects/Player.java [hunk 2]
```diff
@@ -63,27 +94,9 @@ public Player(float xPosition, float yPosition) {
...
-                timeSinceLastFloorContact = 0;
-                speedY = 300;
-                canJump = false;
-                playJumpSound();
-            }
+
+            checkForKeys();
+
             if (respawned > 0) {
                 respawned = respawned - elapsed;
```
Comment (DaanVermunt) [inline core/src/main/java/sem/group15/bubblebobble/core/objects/Player.java]: nice job makink this more readable <layout>

**Comment by DaanVermunt** [comment]
Nice job refactoring this <layout, organization>

**Pull request merged**